.github/workflows/audit.yml

```diff
@@ -0,0 +1,41 @@
+name: Security audit
+
+on:
+  schedule:
+    # Runs at 00:00 UTC everyday
+    - cron: '0 0 * * *'
+  push:
+  pull_request:
+
+jobs:
+  rust-audit:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: EmbarkStudios/cargo-deny-action@3f4a782664881cf5725d0ffd23969fcce89fd868 # v1.6.3
+        with:
+          manifest-path: "deps/swc/bindings/Cargo.toml"
```

Comment:
Oh you are using `swc/bindings`. I'm not sure why it fails.

Reply:
just updated and it works, maybe I can remove everything except the binding folder

Reply:
It should never use path dependency. I think it can be an issue of `EmbarkStudios/cargo-deny-action`.
If you run `cargo deny` from `./bindings` of the swc repository it works well.

Reply:
since I only use binding_typescript_wasm I should be able to remove everything except that, and run cargo deny in that folder